src/pages/patientView/trialMatch/TrialMatchTable.tsx

```diff
@@ -0,0 +1,236 @@
+import * as React from 'react';
+import { If, Then, Else } from 'react-if';
+import {observer} from "mobx-react";
+import {
+    IClinicalGroupMatch, IGenomicGroupMatch, IGenomicMatch, IDetailedTrialMatch, IArmMatch
+} from "../../../shared/model/MatchMiner";
+import styles from './style/trialMatch.module.scss';
+import { computed } from "mobx";
+import LazyMobXTable from "../../../shared/components/lazyMobXTable/LazyMobXTable";
+import SampleManager from "../sampleManager";
+import DefaultTooltip, { placeArrowBottomLeft } from "../../../public-lib/components/defaultTooltip/DefaultTooltip";
+
+export type ITrialMatchProps = {
+    sampleManager: SampleManager | null;
+    detailedTrialMatches: IDetailedTrialMatch[];
+    containerWidth: number;
+}
+
+enum ColumnKey {
+    ID = 'ID',
+    TITLE = 'Title',
+    MATCHING_CRITERIA = 'Matching Criteria'
+}
+
+enum ColumnWidth {
+    ID = 140
+}
+
+class TrialMatchTableComponent extends LazyMobXTable<IDetailedTrialMatch> {
+
+}
+
+@observer
+export default class TrialMatchTable extends React.Component<ITrialMatchProps> {
+
+    @computed
+    get columnsWidth() {
```

Comment:
this is very minor, but it should be "columnWidths", the plural being the widths.

Reply:
Updated